Dockerfile

```diff
@@ -45,7 +45,7 @@ RUN chmod +x /usr/src/{{project_name}}/tasks.py \
     && chmod +x /usr/src/{{project_name}}/entrypoint.sh
 
 # Upgrade pip
-RUN pip install pip==20.1
+RUN pip install pip==20.1.1
```

Comment:
@afabiani just thinking out loud not directly connected to your MR but maybe for future optimization. These four RUN commands do all take care of pip install. We could merge them into one RUN command to not create a new layer within the image ... trying to keep it small:

```
# Install pip packages
RUN pip install pip==20.1.1 \
    pip install --upgrade --no-cache-dir --src /usr/src -r requirements.txt \
    pip install pygdal==$(gdal-config --version).* \
    pip install flower==0.9.4
```

Reply:
Yep, agree